client/src/main/java/org/apache/rocketmq/client/impl/consumer/AssignedMessageQueue.java

```diff
@@ -83,10 +83,15 @@ public long getPullOffset(MessageQueue messageQueue) {
         return -1;
     }
 
-    public void updatePullOffset(MessageQueue messageQueue, long offset) {
+    public void updatePullOffset(MessageQueue messageQueue, long offset, ProcessQueue processQueue) {
         MessageQueueState messageQueueState = assignedMessageQueueState.get(messageQueue);
```

Comment:
If return null here, it may report to NPE.

Reply:
fixed